apps/web/src/views/PositionManagers/hooks/useFilters.ts

```diff
@@ -7,18 +7,26 @@ function filterHookFactory(filterName: string) {
     const router = useRouter()
     const filter = useMemo(() => {
       const value = router.query[filterName]
-      if (Array.isArray(value)) {
-        return value[0]
+      if (value) {
```

Comment:
It will break if I empty the search field?

Reply:
<img width="1887" alt="image" src="https://github.com/pancakeswap/pancake-frontend/assets/116779127/c6e49eba-5fd1-40a2-a02c-d9f9456b60e6">

was trying to delete the search and it's not working